adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala

```diff
@@ -235,7 +235,23 @@ class ADAMContext(@transient val sc: SparkContext) extends Serializable with Log
     }
   }
 
-  def loadBam(filePath: String): RDD[AlignmentRecord] = {
+  /**
+   * Loads a SAM/BAM file.
+   *
+   * This reads the sequence and record group dictionaries from the SAM/BAM file
+   * header. SAMRecords are read from the file and converted to the
+   * AlignmentRecord schema.
+   *
+   * @param filePath Path to the file on disk.
+   *
+   * @return Returns a Tuple3 containing (an RDD of reads, the sequence
+   *   dictionary representing the contigs these reads are aligned to
+   *   if the reads are aligned, the record group dictionary for the reads
+   *   if one is available).
+   *
+   * @see loadAlignments
+   */
+  def loadBam(filePath: String): (RDD[AlignmentRecord], SequenceDictionary, RecordGroupDictionary) = {
```

Comment:
Evidenced by the number of places ```._1``` had to be added, this is a rather breaking change, even though we aren't committed to binary compatibility at this point.

My concern though is more from the story telling point -- the main abstraction in Spark is the RDD and ADAM supports loading those from native bioinformatics file formats, e.g. you load a BAM file and get an RDD of AlignmentRecords.  Except now that's not the method signature because there's some other stuff too.

How is that for a weak argument?  :)